rules/no-assign-parameter.js

```diff
@@ -32,10 +32,11 @@ module.exports = {
         }
 
         function checkExpressionStatement(statement, node, params) {
+            let function_name = node.name ? `"${node.name}"` : 'Fallback function';
```

Comment:
Could you please name it `functionName` instead since mixedCase is the convention throughout the repo

Reply:
Of course - just changed that @duaraghav8